<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanelWindow</class>
 <widget class="QMainWindow" name="PanelWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1640</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1640</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Main Panel | SpiderMatch</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../assets/icon_teal.png</normaloff>../assets/icon_teal.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1640</width>
     <height>900</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QGroupBox" name="config_group_box">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Configuracion</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QTabWidget" name="config_tabs">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>651</width>
         <height>341</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="generalTab">
        <attribute name="title">
         <string>Tab 1</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="generalTab2">
        <attribute name="title">
         <string>Tab 2</string>
        </attribute>
       </widget>
      </widget>
      <widget class="QPushButton" name="start_search_button">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>400</y>
         <width>181</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Comenzar busqueda</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="progress_bar">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>400</y>
         <width>451</width>
         <height>51</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="sites_group_box">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Sitios y Fuentes</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>771</width>
         <height>411</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QListView" name="site_list"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="site_import_button">
            <property name="text">
             <string>Importar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="site_export_button">
            <property name="text">
             <string>Exportar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="site_add_button">
            <property name="text">
             <string>Agregar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="site_delete_button">
            <property name="text">
             <string>Eliminar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="rules_group_box">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Reglas de Busqueda</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>781</width>
         <height>411</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QListWidget" name="rule_list"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="rule_import_button">
            <property name="text">
             <string>Importar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="rule_export_button">
            <property name="text">
             <string>Exportar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="rule_add_button">
            <property name="text">
             <string>Agregar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="rule_delete_button">
            <property name="text">
             <string>Eliminar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="log_group_box">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Registro / Log</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>781</width>
         <height>411</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Exportar</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QScrollArea" name="log_area">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>777</width>
             <height>379</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1640</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Guardar como</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
